NAMESPACE

```diff
@@ -54,6 +54,7 @@ export(logToLin)
 export(metadata)
 export(mixedToLog)
 export(observeSolute)
+export(plotEGRET)
```

Comment:
the MVP *is* `plotEGRET`. agreed that it's potentially complicated, but we're on the hook for some basic, usable plots by next week. my sense is that exposing `convertToEGRET` right now just *has* to be more complicated than not exposing it, all else being equal.